requirements/edx/base.in

```diff
@@ -171,21 +150,48 @@ social-auth-core
 simplejson
 Shapely                             # Geometry library, used for image click regions in capa
 six                                 # Utilities for supporting Python 2 & 3 in the same codebase
-skill-tagging                       # Library needed to support tagging of skills in unit/video xblocks via forms
 social-auth-app-django
 sorl-thumbnail
 sortedcontainers                    # Provides SortedKeyList, used for lists of XBlock assets
 sqlparse                            # Required by Django to run migrations.RunSQL
 stevedore                           # Support for runtime plugins, used for XBlocks and edx-platform Django app plugins
-staff-graded-xblock                 # An XBlock that allows for off-site bulk scoring of problems
-super-csv                           # Generic CSV processor
 unicodecsv                          # Easier support for CSV files with unicode text
 user-util                           # Functionality for retiring users (GDPR compliance)
 webob
 web-fragments                       # Provides the ability to render fragments of web pages
 XBlock[django]                      # Courseware component architecture
 xblock-utils                        # Provides utilities used by the Discussion XBlock
 xss-utils                           # https://github.com/edx/edx-platform/pull/20633 Fix XSS via Translations
+
+########################################################################################################################
+## Non-kernel dependencies
+# The following dependencies do not seem to be core to the platform's functionality, and will soon be moved to a
+# separate "default.in" file. They may still be installed by default, but they should not be required for the platform
+# to run nor for the test suite to pass.
```

Comment:
Do we believe that this list should not grow, and should we note that, or when and how it should grow, and links to documentation for how to avoid growing this list?

Reply:
I'm deferring that decision for now as I believe the Product Working Group is still hashing that out. @kdmccormick please let us know if there have been any decisions made though.

Reply:
Will do. Unfortunately, no update yet.

I imagine that we would like to see both:

* the list of kernel dependencies **not** grow (or only grow very thoughtfully),
* the list of non-kernel dependencies to grow as much it needs to, according to the core product, and
* the list of kernel vs non-kernel dependencies accurately represent what is and isn't required, respectfully, for edx-platform to run.

The [OEP](https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0057-proc-core-product.html#bundled-extensions) spells that out in the abstract, but I don't know if it's concrete enough for every day edx-platform developers. I do think we will want more targeted documentation for adding edx-platform requirements once this effort is further along.